src/main/java/essenmakanan/command/AddRecipeCommand.java

```diff
@@ -16,9 +16,22 @@ public AddRecipeCommand(String toAdd, RecipeList recipes) {
 
     @Override
     public void executeCommand() {
-        String recipeTitle = RecipeParser.parseRecipeTitle(toAdd);
-        Recipe newRecipe = new Recipe(recipeTitle);
-        recipes.addRecipe(newRecipe);
-        ui.printAddRecipeSuccess(recipeTitle);
+        if (toAdd.contains("r/") & toAdd.contains("s/")) {
+            String[] allToAdd = toAdd.split("s/");
+            String recipeTitle = RecipeParser.parseRecipeTitle(allToAdd[0].trim());
```

Comment:
toAdd shouldnt have "r/", if it does, should be removed i parser class. Hence, dont need Recipe.Parser.parseRecipeTitle(...)

Reply:
Actually, toAdd contains "r/", you may refer this screenshot
<img width="615" alt="Screenshot 2023-10-26 at 15 48 34" src="https://github.com/AY2324S1-CS2113-F11-2/tp/assets/139958049/89664aba-bf48-4b34-9d79-ae7dfb77cfae">